airbyte-integrations/connectors/source-youtube-data/manifest.yaml

```diff
@@ -623,6 +615,10 @@ schemas:
                 type:
                   - string
                   - "null"
+              favorites:
```

Comment:
Could you please also update video stream schema to have new fields after removing snippet from record selector. like: etag, id, kind etc.